src/common/Formatter.h

```diff
@@ -3,18 +3,19 @@
 #ifndef CEPH_FORMATTER_H
 #define CEPH_FORMATTER_H
 
-#include "include/int_types.h"
-
 #include <deque>
 #include <iosfwd>
 #include <list>
 #include <vector>
-#include <sstream>
 #include <stdarg.h>
+#include <stddef.h>
+#include <stdint.h>
+#include <utility>
+#include <sstream>
 #include <string>
 #include <map>
 
-#include "include/buffer_fwd.h"
```

Comment:
i think `buffer_fwd.h` is a better choice? as it just forward-declares the buffer types instead of including the full definitions of them, which slow down the compilation.